ui-tests/test/contents.test.ts

```diff
@@ -22,7 +22,7 @@ test.use({
   waitForApplication: firefoxWaitForApplication,
 });
 
-test.describe('Contents Tests', () => {
+test.describe.only('Contents Tests', () => {
```

Comment:
You can also use the `-g` Playwright command line option to only run a subset of the test suite.

Reply:
thanks! nice tip